### ISSUE TITLE
Refactor model code structure

### ISSUE THREAD

**Issue by slifty** [prior-art-archive/overview#26]
We right now have a `models.js` which is fine as a starting point.  It is good practice to have a `models/` directory where each model has its own file.  This is mostly intended to avoid the situation where we have a long monolithic file.

**Comment by isTravis** [comment]
Agreed. We've refactored our PubPub to be structured like this. If you'd like an example of how to deal with Sequelize and associations when split across files - feel free to take a look at the [PubPub /models folder](https://github.com/pubpub/pubpub/tree/v6/server/models). 

Also happy to submit a PR for this if it's not a rush.

**Comment by slifty** [comment]
@isTravis that would be awesome; I'll assign it to you (and if we get to it before you start then I'll coordinate)
